gravitee-apim-console-webui/src/management/home/home-api-status/home-api-status.component.html

```diff
@@ -16,122 +16,149 @@
 
 -->
 
-<gio-table-wrapper
-  [searchLabel]="searchLabel"
-  [length]="apisTableDSUnpaginatedLength"
-  [filters]="filters"
-  (filtersChange)="onFiltersChanged($event)"
->
-  <table mat-table matSort [dataSource]="apisTableDS" id="apisTable" aria-label="Apis table">
-    <!-- Picture Column -->
-    <ng-container matColumnDef="picture">
-      <th mat-header-cell *matHeaderCellDef id="picture"></th>
-      <td mat-cell *matCellDef="let element">
-        <gio-avatar [src]="element.picture" [name]="element.name" size="32" [roundedBorder]="true"></gio-avatar>
-      </td>
-    </ng-container>
+<gio-banner-info>
+  Health monitoring of your APIs by sending requests to an health endpoint on each API. The API backend should perform the necessary checks
+  by providing some probes and return an indication of its status.
```

Comment:
I feel this sentence could be rephrased. Maybe @jourdiw could help.

Reply:
Health-checks are performed on your APIs by interrogating each API's health endpoint. After receiving a health-check request, each API backend will perform the necessary checks via probes to return its collective status.